File: jul2020/Zadatak/Model/Match.cs
using System;

namespace Zadatak.Models
{
    public class Match
    {
        public int Id { get; set; }
        public Player Player1 { get; set; }

        public Player Player2 { get; set; }

        public string Location { get; set; }

        public DateTime Time { get; set; }

        public Result Result { get; set; }
    }
}

File: jul2020/Zadatak/Javascript/match.js
import { Result } from "./result";

export class Match
{
    constructor(container, player1, player2, time, date)
    {
        this.container = container;
        this.player1 = player1;
        this.player2 = player2;
        this.time = time;
        this.date = date;
        this.result = null;
        this.draw();
    }

    draw()
    {

    }
}